### PR TITLE
stdevOsmel t4 pincode backspace

### PR DESCRIPTION
**Pull request by stdevAlDen**
Added backspace for pingcode keyboard

### CODE REVIEW

File: src/pages/pincode/pincode.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { Component, ElementRef } from '@angular/core';
```
Comment (samuelvisscher) [inline src/pages/pincode/pincode.ts:1]: Please leave existing bracket spacing

File: src/pages/pincode/pincode.ts [hunk 2]
```diff
@@ -21,12 +21,10 @@ export class PincodePage {
   storageAvailable = true;
 
 
-  constructor(
```
Comment (samuelvisscher) [inline src/pages/pincode/pincode.ts]: Please leave existing indentation

**Review by samuelvisscher — CHANGES_REQUESTED** [review]

File: src/pages/pincode/pincode.ts (hunk 1 above)
Comment (stdevAlDen) [reply src/pages/pincode/pincode.ts:1]: resolved

File: src/pages/pincode/pincode.ts (hunk 2 above)
Comment (stdevAlDen) [reply src/pages/pincode/pincode.ts]: resolved

**Comment by samuelvisscher** [comment]
Excellent, could you please make this PR against dev?

**Comment by stdevGeekmidget** [comment]
Please close this PR, we do not need it in develop branch!!

**Pull request closed without merging**